.pre-commit-hooks.yaml

```diff
@@ -4,6 +4,12 @@
     entry: '(?i)# noqa(?!: )'
     language: pygrep
     types: [python]
+-   id: python-check-blanket-nosec
+    name: check blanket nosec
+    description: 'Enforce that bandit `nosec` annotations always occur with specific codes. Sample annotations: `# nosec: B101`, `# nosec: B101,B102`'
+    entry: '# *nosec(?!: *\w)'
```

Comment:
is there a reason this is different from the noqa pattern?

Reply:
It is to have something similar to bandit regex (https://github.com/mikespallino/bandit/blob/5747e306262d71aca46542f71c6fda116de98b6c/bandit/core/manager.py#L25), though for consistency it may indeed be nice to have something closer to `noqa` pattern (maybe without `(?i)`, as `nosec` is case sensitive).